SYCL/Basic/alloc_on_subdevice.cpp

```diff
@@ -0,0 +1,33 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: env CreateMultipleSubDevices=2 NEOReadDebugKeys=1 SYCL_DEVICE_FILTER="gpu" %t.out
+// UNSUPPORTED: gpu-intel-dg1
+
+// REQUIRES: gpu

```

Comment:
```suggestion
// REQUIRES: gpu
```suggestion
// REQUIRES: gpu, level_zero

```
```